src/components/common/Admin/AdminHeader.js

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { useAuth0 } from "@auth0/auth0-react";
+import "./styles/Header.css";
+// import { storySquadLogo } from "./utils/pics";
+import {useNavigate} from "react-router-dom";
+
+const Header = () => {
+    // const { user, isAuthenticated, logout } = useAuth0();
+    // authentication will be handled by Auth0 //
+    const navigate = useNavigate();
+  
+    return (
+        // isAuthenticated && ( checks for athenticated user 
```

Comment:
I'm sure this is just temporary code that will be used once authentication is incorporated. May need to remove until then.

Reply:
I've uncommented this piece of code since the project does have Auth0 installed so I think there should be no issues